packages/contracts/src/contracts/current/protocol/Exchange/IExchange.sol

```diff
@@ -124,86 +124,86 @@ contract IExchange {
         returns (bool isValid);
 
     /// @dev Fills the input order.
-    /// @param orderAddresses Array of order's maker, taker, makerToken, takerToken, and feeRecipient.
-    /// @param orderValues Array of order's makerTokenAmount, takerTokenAmount, makerFee, takerFee, expirationTimestampInSec, and salt.
-    /// @param takerTokenFillAmount Desired amount of takerToken to fill.
+    /// @param orderAddresses Array of order's maker, taker, makerAsset, takerAsset, and feeRecipient.
+    /// @param orderValues Array of order's makerAssetAmount, takerAssetAmount, makerFee, takerFee, expirationTimestampInSec, and salt.
+    /// @param takerAssetFillAmount Desired amount of takerAsset to fill.
     /// @param v ECDSA signature parameter v.
     /// @param r ECDSA signature parameters r.
     /// @param s ECDSA signature parameters s.
-    /// @return Total amount of takerToken filled in trade.
+    /// @return Total amount of takerAsset filled in trade.
     function fillOrder(
           address[5] orderAddresses,
           uint256[6] orderValues,
-          uint256 takerTokenFillAmount,
+          uint256 takerAssetFillAmount,
```

Comment:
Are we not yet passing an Order struct into the `fillOrder` function?

Reply:
We just haven't updated this interface yet since things aren't quite final yet.